nipype/interfaces/afni/preprocess.py

```diff
@@ -2618,6 +2618,10 @@ class TShiftInputSpec(AFNICommandInputSpec):
         desc='time offsets from the volume acquisition onset for each slice',
         argstr='-tpattern @%s',
         xor=['tpattern'])
+    slice_encoding_direction = traits.Enum(
+        ('k', 'k-'),
+        desc='Direction in which slice_timing is specified (default: k). If negative,'
+             'slice_timing is defined in reverse order -- see BIDS specification for details.',)
```

Comment:
We can have `k` be default as follows:

```Python
traits.Enum('k', 'k-', usedefault=True, desc=...)
```

Also, I would not refer to the BIDS spec here, but just explain that `k` means the third spatial dimension of the data block. While it's nice to be consistent with BIDS, when possible, nipype should stand on its own. (You can copy-paste from the spec, if you like, however.) And I would note that it only has an effect if `slice_timing` is a list; we will not reorder files.